src/components/AssignmentCreatorForm.jsx

```diff
@@ -69,43 +70,61 @@ export default function AssignmentCreatorForm({
           valueLabel={parsedDate}
         />
       </div>
-      <button
-        className={classnames(
-          'assignment-creator__button',
-          'assignment-creator__button_reject',
-        )}
-        type="button"
-        onClick={onCloseAssignmentCreator}
-      >
-        {t('assignment-creator.cancel-button')}
-      </button>
-      <button
-        className={classnames(
-          'assignment-creator__button',
-          'assignment-creator__button_confirm',
-        )}
-        type="button"
-        onClick={handleSubmit(onDraftAssignment)}
-      >
-        {t('assignment-creator.draft-button')}
-      </button>
-      <button
-        className={classnames(
-          'assignment-creator__button',
-          'assignment-creator__button_confirm',
-        )}
-        type="button"
-        onClick={handleSubmit(onAssignAssignment)}
-      >
-        {t('assignment-creator.assign-button')}
-      </button>
+      {
+        isAssignmentExportInProgress ?
+          <div>
+            <button
+              disabled
+              className={classnames(
+                'assignment-creator__button',
+                'assignment-creator__button_disabled',
+              )}
+              type="button"
+            >
+              {t('assignment-creator.creating')}
+            </button>
+          </div> :
+          <div>
```

Comment:
You can use a [fragment](https://reactjs.org/docs/fragments.html) here! No need for the extra `<div>` just for JSX structural reasons.